### ISSUE TITLE
[Feature Request / Bug]: autoscaler_nodepools example labels is wrong

### ISSUE THREAD

**Issue by dstockhammer** [kube-hetzner/terraform-hcloud-kube-hetzner#1074]
### Description

The latest example values for `labels` and `taints` in `autoscaler_nodepools` in `kube.tf.example` seem wrong:

```tf
autoscaler_nodepools = [
  {
    # ...
    labels      = {
      "node.kubernetes.io/role=peak-workloads"
    }
    taints      = 
    [{
       key: "node.kubernetes.io/role"
       value: "peak-workloads"
       effect: "NoExecute"
    }]  
  }
]
```

From looking at the code I believe that it should be the following:

```tf
autoscaler_nodepools = [
  {
    #...

    labels = [{
      "node.kubernetes.io/role": "peak-workloads"
    }]

    taints = [{
        key: "node.kubernetes.io/role"
        value: "specific-workloads"  # not `peak-workloads`
        effect: "NoExecute"
    }]
  }
]
```

However, now the labels and taints schemas are different for `autoscaler_nodepools` and `agent_nodepools`. I'm not convinced that this makes a lot of sense...

Is the plan to change all labels/taints to the new more verbose schema? Obviously as maintainers you're free to choose whichever schema you wish (although I personally prefer the original more concise string over the object), but in either case I think it should be consistent.

Thanks :)

**Comment by Silvest89** [comment]
The new schema allows for easier implementation in the cluster-autoscaler codebase. (I had a pr merged for that as well) 
Kubernetes labels are maps, and Taints are objects. So it was easier to make them more in line with Kubernetes.

But yeah I will open a PR to fix the wrong example :P

**Comment by mysticaltech** [comment]
@dstockhammer Please terraform upgrade -init to the latest version and use the new schema by @Silvest89. See latest kube.tf.example.

**Comment by mysticaltech** [comment]
@dstockhammer The fix by @Silvest89 was just released in v2.9.1. Enjoy!
